syntax = "proto3";

package store;

import "empty.proto";

service HealthCheck {
    rpc Check (Empty) returns (Empty);
}